src/scripts/main.js

```diff
@@ -1,3 +1,369 @@
 'use strict';
 
-// write your code here
+const startButton = document.querySelector('.start');
+const message = document.querySelector('.message-container');
+
+const fieldSide = 4;
+const fieldRows = [...document.querySelectorAll('tr')];
+const fieldCells = [...document.querySelectorAll('td')];
+const score = document.querySelector('.game-score');
+
+function generateInEmptyCell() {
+  do {
+    const x = Math.floor(Math.random() * fieldSide);
+    const y = Math.floor(Math.random() * fieldSide);
+
+    if (fieldRows[x].children[y].textContent === '') {
```

Comment:
Move long expressions to variable. Fix all cases please
```suggestion
    if (!fieldRows[x].children[y].textContent) {
```